### PR TITLE
abranch8: Testing generic support comments

### PR DESCRIPTION
**Pull request by gudmdharalds**

### CODE REVIEW

File: bla-4.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo "foo" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-4.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

**Comment by gudmdharalds** [comment]
test

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

File: bla-3.php [hunk 2]
```diff
@@ -1,3 +1,8 @@
 <?php
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: :information_source: **Info**: File is approved in review database (hashes-to-hashes) (*WordPressVIPMinimum.Info.ApprovedHashesToHashesAPI*).

Comment (gudmdharalds-a8c) [inline bla-3.php]: :information_source: **Info**: File is approved in review database (hashes-to-hashes) (*WordPressVIPMinimum.Info.ApprovedHashesToHashesAPI*).

File: bla-4.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo "testing " . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-4.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-3.php (hunk 2 above)
Comment (gudmdharalds-a8c) [inline bla-3.php]: :information_source: **Info**: File is approved in review database (hashes-to-hashes) (*WordPressVIPMinimum.Info.ApprovedHashesToHashesAPI*).

**Comment by gudmdharalds-a8c** [comment]
This Pull Request has been added to the VIP review queue and will be reviewed and updated by a member of the VIP code review team (as per your SLA). In the meantime, please review and respond to any automated feedback generated by the @wpcomvip-vipgoci-bot.<br><br> If you’d like to request an urgent VIP review, please add the VIP urgent review label or reply to this PR with a comment containing the word <code>urgent</code> to notify the entire VIP code review team.

File: bla-3.php [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
 
 echo "testing " . time() . PHP_EOL;
+
+/* Comments */
+/* More comments */
+
+   // Whitespace and comments
+
+echo  "testing " . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-3.php [hunk 5]
```diff
@@ -1,3 +1,19 @@
 <?php
 
-echo "testing " . time() . PHP_EOL;
+echo "testing   " . time() . PHP_EOL;
+echo "testing   " . time() . PHP_EOL;
+echo "testing   " . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-3.php [hunk 6]
```diff
@@ -1,3 +1,19 @@
 <?php
 
-echo "testing " . time() . PHP_EOL;
+echo "testing   " . time() . PHP_EOL;
+echo "testing   " . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: bla-3.php [hunk 7]
```diff
@@ -1,3 +1,19 @@
 <?php
 
-echo "testing " . time() . PHP_EOL;
+echo "testing   " . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Pull request closed without merging**